sdk/src/wallet/operations/transaction/build_transaction.rs

```diff
@@ -49,7 +48,6 @@ where
             }
 
             inputs.push(Input::Utxo(UtxoInput::from(*input.output_id())));
-            inputs_for_signing.push(input.clone());
```

Comment:
Was there any reason to do this?

Reply:
Checked the history and no, I probably have overseen that it can be used directly